src/util.ts

```diff
@@ -49,3 +49,8 @@ export function uuid() {
     }
   }).join("")
 }
+
+export function reportError(error) {
+  console.warn(error)
```

Comment:
Shouldn't be this console.error to stay true to the method name and the old function?